src/Extensions/LDAPMemberExtension.php

```diff
@@ -264,15 +283,22 @@ public function sync()
     }
 
     /**
-     * Triggered by {@link Member::logIn()} when successfully logged in,
-     * this will update the Member record from AD data.
+     * Triggered by {@link IdentityStore::logIn()}. When successfully logged in,
+     * this will update the Member record from LDAP data.
      */
-    public function memberLoggedIn()
+    public function afterMemberLoggedIn()
```

Comment:
I guess this is technically a semver break, even if the old method name was wrong to start with. Maybe we could keep the old one and `return $this->afterMemberLoggedIn()` with a deprecation notice on it?

Reply:
Thanks for doing this @robbieaverill, I missed that when upgrading!